ui/imports/shared/stores/RootStore.qml

```diff
@@ -71,6 +72,18 @@ QtObject {
     }
 
     function copyToClipboard(textToCopy) {
-        chatsModelInst.copyToClipboard(textToCopy);
+        chatsModelInst.copyToClipboard(textToCopy)
```

Comment:
@alexandraB99 this `chatsModelInst ` refers to the `chatsModel` which doesn't exists in the app, use `globalUtils. copyToClipboard(textToCopy)` instead.